python-threatexchange/threatexchange/cli/experimental_fetch.py

```diff
@@ -76,7 +76,9 @@ def __init__(
         self.processed = 0
         self.counts = collections.Counter()
 
-    def execute(self, api: ThreatExchangeAPI, dataset: Dataset) -> None:
+    def execute(
+        self, api: ThreatExchangeAPI, dataset: Dataset
+    ) -> None:
```

Comment:
```suggestion
```suggestion
    def execute(self, api: ThreatExchangeAPI, dataset: Dataset) -> None:
```
```